source/__snapshots__/default-highway-test.snap

```diff
@@ -12,7 +11,7 @@ export interface Dashboard {
   is_shared_with_everyone?: boolean;
   name?: string;
   widgets?: DashboardWidget[];
-  __entity_type__?: "Dashboard";
+  __entity_type__: "Dashboard";
```

Comment:
I'm not sure we can guarantee that the `__entity_type__` is always there. But maybe.

Reply:
In our tests, we have never encountered a scenario where it wasn't present. But that doesn't mean it doesn't exist.

Reply:
I haven't seen one either, but I've not gotten any commitments that we can always say it's there.
I'll try to get some answers on that.